pkg/cmd/openshift-tests/run-disruption/disruption.go

```diff
@@ -54,122 +59,183 @@ func NewRunInClusterDisruptionMonitorCommand(ioStreams genericclioptions.IOStrea
 		SilenceUsage:  true,
 		SilenceErrors: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return disruptionOpt.Run()
+			ctx, cancelFn := context.WithCancel(context.Background())
+			defer cancelFn()
+			abortCh := make(chan os.Signal, 2)
```

Comment:
move signal handling to the top to allow embedding the Run if anotehr command wants to.